content/variables/environment-variable-groups.md

```diff
@@ -28,6 +28,32 @@ workflows:
 
 Variables defined in environment variable groups work exactly as [Environment Variables](../building/environment-variables/#using-environment-variables). The value of a variable named `API_TOKEN` can be referenced in a workflow as `$API_TOKEN`. Variables defined with the **_secure_** option will have values obfuscated in the Codemagic UI.
 
+## Encrypting values
+
+Putting values in the Variable value input and marking "Secure" checkbox will automatically encode those values. However, encrypting files requires an additional step and the process is done with the help of different machine specific command lines.
```

Comment:
"Secure" box doesn't encode the values, it encrypts them.

The encoding has to be done by the user (if necessary).

Thus the flow is to:
a) base64 encode string/file
b) paste it into the box
c) click secure

now the value is both base64 encoded and encrypted

Reply:
thanks for explanation, changes made

Reply:
Info wise okay, I guess, though I'd ask @helinanever to review wording etc.